app/javascript/components/Story/Story.tsx

```diff
@@ -108,7 +108,7 @@ const HighlightURL: React.FC<{
   queryID: string;
   indexName: string;
 }> = ({ hit: { url, _highlightResult, objectID }, queryID, indexName }) => {
-  const highlighted = `(${_highlightResult.url.value})`;
+  const highlighted = _highlightResult.url.value;
```

Comment:
Would there be an appropriate way to eliminate parenthesis on mobile only? Because this also eliminates them in desktop.

Reply:
Decided to remove this change from the PR